packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -287,6 +290,17 @@ const serviceEntityPage = (
       </Grid>
     </EntityLayout.Route>
 
+    <EntityLayout.Route path="/dependencies" title="Dependencies">
+      <Grid container spacing={3} alignItems="stretch">
+        <Grid item md={6}>
+          <EntityDependsOnComponentsCard />
+        </Grid>
+        <Grid item md={6}>
+          <EntityDependsOnResourcesCard />
```

Comment:
Repeat question regarding `gridItem` variants?

Again, seems pretty duplicative to have to put it everywhere, but it seems to be a common pattern in here?